### PR TITLE
Kh create models

### PR DESCRIPTION
**Pull request by kheppenstall**
Create admin, issue, photo, and comment models based on schema.

@ski-climb

### CODE REVIEW

File: app/models/admin.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Admin < ApplicationRecord
+  has_secure_password
+  has_many :issues
+  has_many :comments
+  has_many :photos
```
Comment (ski-climb) [inline app/models/admin.rb]: @kheppenstall Nice catch on the photos!  I forgot the `admin_id` column on the `photos` table yesterday.

File: spec/factories/comments.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :comment do
+    issue
+    user
+    admin nil
```
Comment (ski-climb) [inline spec/factories/comments.rb]: `admin nil` - that's handy.  I didn't know you could do that in Factory Girl.

File: spec/factories/issues.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :issue do
+    user
+    admin       nil
+    title       { Faker::Commerce.department }
+    description { Faker::Hipster.paragraph }
+    category    { rand(4) }
+    severity    { rand(2) }
+    latitude    { Faker::Address.latitude }
```
Comment (ski-climb) [inline spec/factories/issues.rb]: Nice!   The lat/lon even come in as floats - no converting from decimal.  Nice find.

File: spec/models/issue_spec.rb [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    it 'successfully saves to the database' do
+      issue = create(:issue)
+      saved_issue = Issue.first
+
+      expect(Issue.count).to eq 1
+      expect(saved_issue.title).to eq issue.title
+    end
+  end
+
+  context 'relationships' do
```
Comment (ski-climb) [inline spec/models/issue_spec.rb]: @kheppenstall What would you think about moving lines 37 and 38 up to be near the `have_many` expectations on lines 22, 23?  That way all of the 'relationships' specs are in the same place/block.

File: spec/models/issue_spec.rb [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+      expect(issue.medium?).to be_truthy
+      expect(issue.severity).to eq 'medium'
+    end
+
+    it '#high' do
+      issue = create(:issue, severity: 2)
+
+      expect(issue.high?).to be_truthy
+      expect(issue.severity).to eq 'high'
+    end
```
Comment (ski-climb) [inline spec/models/issue_spec.rb]: @kheppenstall Nice and thorough on the tests for `severity` and `category`.  👍

File: spec/models/issue_spec.rb (hunk 4 above)
Comment (kheppenstall) [inline spec/models/issue_spec.rb]: @ski-climb Nice catch. Not sure what I was doing there.

File: app/models/admin.rb (hunk 1 above)
Comment (kheppenstall) [inline app/models/admin.rb]: 👍

**Comment by kheppenstall** [comment]
@ski-climb Fixed the double relationship block in `issue_spec`

**Pull request merged**